code/__DEFINES/mobs.dm

```diff
@@ -11,6 +11,9 @@
 #define MOVE_INTENT_WALK "walk"
 #define MOVE_INTENT_RUN  "run"
 
+//Walking Speed Threshold for slips and stuff. If you move slower than this, you are considered walking, regardless of intent.
+#define MOVE_WALK_THRESHOLD 4 // Roundstart with shoes in walk-intent your move-delay reads as 4.
+
```

Comment:
Wouldn't it make more sense to have this == the actual slowdown applied by walk intent?

Reply:
Yea i wanted to ask:

Should i do: 
#define MOVE_WALK_THRESHOLD CONFIG_GET(WALK_DELAY)

or should i get rid of MOVE_WALK_THRESHOLD entirely and just use CONFIG_GET(WALK_DELAY)

Reply:
latter

Reply:
Done